src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
I know, the adoption rate of python 3 has been depressing. But having shorah in bioconda helps this and the dependency on future.

I would remove global analysis while porting the current code to pure python 3, then add python 2 back compatibility. I'm not familiar with the module future, but I see in the [quick-start](http://python-future.org/quickstart.html) that this amounts to add

    from __future__ import (absolute_import, division,
                            print_function, unicode_literals)
    from builtins import *

I was confused by the `from future.utils import listitems` statement.